packages/vue-table/package.json

```diff
@@ -42,8 +42,12 @@
     "build/umd/*",
     "src"
   ],
+  "scripts": {
+    "clean": "rimraf ./build",
+    "test:types": "tsc"
+  },
   "dependencies": {
-    "@tanstack/table-core": "8.11.2"
+    "@tanstack/table-core": "workspace:*"
```

Comment:
Doesn't this need to tied to a version?

Reply:
This gets installed in a user's node_modules

Reply:
Pnpm will handle replacing workspace fields with versions during publishing - you can see more about it here: https://pnpm.io/workspaces#publishing-workspace-packages. And whatever files are in node_modules would be symlinks back to the local folder :)